db/migrations/20221117154600-add-new-clientn-key-column-to-installation-table.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn("Installations", "plainClientKey", { type: Sequelize.STRING, allowNull: true });
```

Comment:
name suggestion welcome. 
As `models/Installation` has `clientKey`, and `models/Subscription` has `jiraClientKey`. I am out of options.
I want to make variable clearly distinguishable ( at least before we deprecate the hashed client key)